release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1275,25 +1280,40 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |Technology Preview
 
+|{sno-caps} cluster expansion with worker nodes
+| General Availability
+| General Availability
+| General Availability
+
+|{cgu-operator-first}
+|General Availability
+|General Availability
+|General Availability
+
+|NUMA-aware scheduling with NUMA Resources Operator
+|General Availability
+|General Availability
+|General Availability
+
 |Mount namespace encapsulation
 |Technology Preview
 |Technology Preview
-|Technology Preview
+|General Availability
 
 |HTTP transport replaces AMQP for PTP and bare-metal events
 |Technology Preview
 |Technology Preview
-|Technology Preview
+|General Availability
 
 |Intel E810 Westport Channel NIC as PTP grandmaster clock
 |Technology Preview
 |Technology Preview
-|Technology Preview
+|General Availability
 
 |Workload partitioning for three-node clusters and standard clusters
 |Technology Preview
 |Technology Preview
```

Comment:
In https://docs.openshift.com/container-platform/4.14/release_notes/ocp-4-14-release-notes.html#ocp-4-14-technology-preview, 'Workload partitioning for three-node clusters and standard clusters' is already shown as 'General Availability'.

Reply:
Thank you @qiliRedHat ! I will correct this!